assets/src/edit-story/components/panels/design/preset/test/colorPresetPanel.js

```diff
@@ -127,6 +128,8 @@ describe('Panels/Preset', () => {
     return els.length > 0 && els.every(({ type }) => elType === type);
   });
 
+  getPanelInitialHeight.mockImplementation(() => 150);
```

Comment:
FYI, you can also just do [`mockReturnValue(150)`](https://jestjs.io/docs/en/mock-function-api#mockfnmockreturnvaluevalue) for static returns like this.